airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/cdc/MongoDbCdcInitializer.java

```diff
@@ -80,8 +79,14 @@ public List<AutoCloseableIterator<AirbyteMessage>> createCdcIterators(
                                                                         final Instant emittedAt,
                                                                         final MongoDbSourceConfig config) {
 
-    final Duration firstRecordWaitTime = RecordWaitTimeUtil.getFirstRecordWaitTime(config.rawConfig());
-    final Duration subsequentRecordWaitTime = RecordWaitTimeUtil.getSubsequentRecordWaitTime(config.rawConfig());
+//    LOGGER.info("*** config {}", config.rawConfig());
+    final Duration firstRecordWaitTime = Duration.ofSeconds(config.getInitialWaitingTimeSeconds());
+    // #35059: debezium heartbeats are not sent on the expected interval. this is a worksaround to allow
+    // making
+    // subsequent wait time configurable.
+    // final Duration subsequentRecordWaitTime = firstRecordWaitTime.dividedBy(2);
+    final Duration subsequentRecordWaitTime = firstRecordWaitTime;
+    LOGGER.info("*** subs {} init {}", subsequentRecordWaitTime, firstRecordWaitTime);
```

Comment:
Can we have a log line here like : "Subsequent record wait time is : {}" to make it more clear?